docs/source/explanation/algorithms/classification/multi_class_classification.rst

```diff
@@ -4,6 +4,8 @@ Multi-class Classification
 Multi-class Classification is the problem of classifying instances into one of two or more classes. We solve this problem in a common fashion, based on the feature extractor backbone and classifier head that predicts the distribution probability of the categories from the given corpus.
 For the supervised training we use the following algorithms components:
 
+.. _mcl_cls_supervised_pipeline:
```

Comment:
Plus, I'm curious if there's a section to explain the sampler we used for incremental solution. If not, I'd like to add here. Especially for multi-class classification it uses specific balanced sampler.

Reply:
Again, why do we do it silently in the code? I would rather write it in the config
![image](https://user-images.githubusercontent.com/68103869/216465839-07e0a084-4cdd-4700-ae42-2c2d5ea52027.png)

Reply:
I added description about balanced sampler. Please, check it, because I don't know exactly what it is doing. I took info from doc string

Reply:
TaskAdaptHook needs some dynamically changing args such as new or old classes which can be checked in code. So we re-configure it in stage.py again. But as you said, we can register TaskAdaptHook in config just with base args. Looks like we need some refactoring again.